news/1545.plugin

```diff
@@ -0,0 +1 @@
+Add fault tolerance via `max_failure_rate` parameter
```

Comment:
What I mean is to move this file into the news directory inside the plugin itself.

`plugins/hydra_nevergrad_sweeper/news`.